doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -274,9 +274,13 @@ simply :g:`t=u` dropping the implicit type of :g:`t` and :g:`u`.
       .. exn:: Too few occurrences.
          :undocumented:
 
-   .. tacv:: change @term {? {? at {+ @natural}} with @term} in @ident
+   .. tacv:: change @term {? {? at {+ @natural}} with @term} in @goal_occurrences
 
-      This applies the :tacn:`change` tactic not to the goal but to the hypothesis :n:`@ident`.
+      In the presence of :n:`with`, this applies :tacn:`change` to the
+      occurrences specified by :token:`@goal_occurrences`. In the
+      absense of :n:`with`, :token:`@goal_occurrences` is expected to
```

Comment:
```suggestion
      occurrences specified by :token:`goal_occurrences`. In the
      absence of :n:`with`, :token:`goal_occurrences` is expected to
```

This will fix your build.  `:token:` doesn't use `@` unlike `:n:`

Reply:
Thanks for the fix. I use ``:n:`@goal_occurrences` `` then. Should be ok, no?

Reply:
Yup.